bin/queryserver.py

```diff
@@ -119,7 +119,7 @@
 
 # The command is run through subprocess so environment variables are automatically inherited
 java_cmd = '%(java)s -cp ' + hbase_config_path + os.pathsep + hadoop_config_path + os.pathsep + \
-    phoenix_utils.phoenix_queryserver_jar + os.pathsep + phoenix_utils.phoenix_client_jar + \
+    phoenix_utils.phoenix_client_jar + \
```

Comment:
those classes are part of  client jar,  since it uses calcite which is shaded in client jar. Or  we need to bundle that package into queryserver jar.